src/main/java/pro/sky/StreamAPIAndOptional/EmployeeServiceImpl.java

```diff
@@ -0,0 +1,55 @@
+package pro.sky.StreamAPIAndOptional;
+import org.springframework.stereotype.Service;
+import pro.sky.StreamAPIAndOptional.exceptions.EmployeeAlreadyAddedException;
+import pro.sky.StreamAPIAndOptional.exceptions.EmployeeNotFoundException;
+import pro.sky.StreamAPIAndOptional.exceptions.EmployeeStorageIsFullException;
+
+import java.util.*;
+
+@Service
+public class EmployeeServiceImpl implements EmployeeService{
+    private static final int MaxSize =10;
```

Comment:
```suggestion
    private static final int MAX_SIZE = 10;
```
когда static И final надо называть в верхнем регистре